helita/sim/bifrost.py

```diff
@@ -427,54 +638,161 @@ def get_quantity(self, quant, *args, **kwargs):
         - MODULE_QUANT: allows to calculate the module of any vector.
                         It must start with 'mod' followed with the root
                         letter of varname, e.g., 'modb'
-        - DIV_QUANT: allows to calculate the divergence of any vector.
-                     It must start with div followed with the root letter
-                     of the varname, e.g., 'divb'
+        - GRADVECT_QUANT: allows to calculate decompose the vector field.
+                     i.e., it calculates the divergence, rotation and shear.
+                     It must start with div, rot or she followed with the
+                     root letter of the varname, e.g., 'divb' or 'rotbx'
         - SQUARE_QUANT: allows to calculate the squared modules for any
                         vector. It must end with 2 after the root lelter
                         of the varname, e.g. 'u2'.
         """
         quant = quant.lower()
+        DERIV_DESC = 'Spatial derivative (Bifrost units). It must start \n' + \
```

Comment:
Need to add a description of ALL new options in the docstring. Also, I don't like the direction this function is going (too much stuff in one), and the behaviour should be more clear to the user. Maybe add example usage cases in docstring.